src/test/kotlin/com/terraformation/backend/tracking/model/PlantingZoneModelTest.kt

```diff
@@ -300,20 +292,20 @@ class PlantingZoneModelTest {
                           id = 1,
                           plots =
                               listOf(
-                                  monitoringPlotModel(10, permanentCluster = 1),
-                                  monitoringPlotModel(11, permanentCluster = 1),
-                                  monitoringPlotModel(12, permanentCluster = 1),
-                                  monitoringPlotModel(13, permanentCluster = 1),
+                                  monitoringPlotModel(10, permanentIndex = 1),
+                                  monitoringPlotModel(11, permanentIndex = 1),
+                                  monitoringPlotModel(12, permanentIndex = 1),
+                                  monitoringPlotModel(13, permanentIndex = 1),
                                   monitoringPlotModel(14),
                                   monitoringPlotModel(15))),
                       plantingSubzoneModel(
                           id = 2,
                           plots =
                               listOf(
-                                  monitoringPlotModel(20, permanentCluster = 2),
-                                  monitoringPlotModel(21, permanentCluster = 2),
-                                  monitoringPlotModel(22, permanentCluster = 2),
-                                  monitoringPlotModel(23, permanentCluster = 2),
+                                  monitoringPlotModel(20, permanentIndex = 2),
+                                  monitoringPlotModel(21, permanentIndex = 2),
+                                  monitoringPlotModel(22, permanentIndex = 2),
+                                  monitoringPlotModel(23, permanentIndex = 2),
```

Comment:
Same note on clusters for these ones

Reply:
Oops, sloppy of me not to check for more of these. Fixed now, and I've verified there aren't any others in this test.